language/stdlib/transaction_scripts/rotate_authentication_key.move

```diff
@@ -34,12 +34,19 @@ fun rotate_authentication_key(account: &signer, new_key: vector<u8>) {
 }
 spec fun rotate_authentication_key {
     use 0x1::Signer;
+    use 0x1::Errors;
+
     let account_addr = Signer::spec_address_of(account);
     include LibraAccount::ExtractKeyRotationCapabilityAbortsIf;
     let key_rotation_capability = LibraAccount::spec_get_key_rotation_cap(account_addr);
     include LibraAccount::RotateAuthenticationKeyAbortsIf{cap: key_rotation_capability, new_authentication_key: new_key};
 
     /// This rotates the authentication key of `account` to `new_key`
     include LibraAccount::RotateAuthenticationKeyEnsures{addr: account_addr, new_authentication_key: new_key};
+
+    aborts_with [check]
+        Errors::INVALID_STATE,
+        Errors::INVALID_ARGUMENT,
+        Errors::NOT_PUBLISHED; // TODO: Prover cannot prove `account` has LibraAccount published.
```

Comment:
Same here as the ed25519 script: the prologue is checking the existence of `account` so I think this is fine to leave out.

Reply:
Removed this error code by adding a "requires" clause.